src/neo-vm/ExecutionEngine.cs

```diff
@@ -37,6 +37,8 @@ public class ExecutionEngine : IDisposable
         /// </summary>
         public virtual uint MaxInvocationStackSize => 1024;
 
+        public virtual uint MaxTryNestingDepth => 16;
```

Comment:
Should we add a MaxTryNestingDepth for InvocationStack or CurrentContext?

Reply:
What do you mean?

Reply:
I think maybe we can add a MaxTryNestingDepth for CurrentContext instead of InvocationStack, as there could be situations that recursions occuring within TRY

Reply:
The max depth of `InvocationStack` is `1024`. If you allow `16` items in each `ExecutionContext.TryStack`, there will be at most `1024 * 16` items. Do you think it's okay?

Reply:
That's 208 KB's RAM.. But if we limit overall numbers of TRY to 16, it will mean that recursions will also be limited to or less than 16 if the recursions happen within TRY, which is much less than `MaxInvocationStackSize `. I think MaxTryNestingDepth should not be stricter than MaxInvocationStackSize.